Simulations/grids.yaml

```diff
@@ -18,6 +18,13 @@ sources:
     description: ICON LES horizontal grid of DOM02 (EUREC4A_PR1250m_DOM02)
```

Comment:
This description could be made more informative for outsiders and/or a reference provided.

Reply:
Thanks! I made the description a bit more explicit now.